src/generic/processing-notification/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Badge, Icon } from '@edx/paragon';
+import { Settings as IconSettings } from '@edx/paragon/icons';
+import { capitalize } from 'lodash';
+
+import { NOTIFICATION_MESSAGES } from '../../constants';
+
+const ProcessingNotification = ({ isShow, title }) => (
```

Comment:
rename `title` to `message`, it can be done next your PR

Reply:
in legacy this text is a `title`, it even lies in `h2` with the `title` class